CHANGELOG.md

```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
-### [2024-05-08]
+## [Unreleased]
+
+### Added
+
+* [New ParkAPI sources](https://github.com/ParkenDD/parkapi-sources-v3/blob/main/CHANGELOG.md#040)
```

Comment:
These links will eventually break. We should make sure to use tag- or commit-based permalinks.